web-common/src/components/data-graphic/elements/GraphicContext.svelte

```diff
@@ -94,7 +95,12 @@ for any of its children.
       marginBuffer,
       id,
     }),
-  };
+    // FIXME: this cast is not provably safe,
+    // but we've been treating this as if it's true
+    // for at least 19 months, so hopefully it's safe enough.
+    // In any case, this cascading context store approach is
+    // ridiculously over engineered and needs to be rethought
+  } as SimpleDataGraphicConfigurationArguments;
```

Comment:
Just to add, this is most probably not a safe cast. I remember seeing missing properties at couple of places some time back. We should get back to this when we have more bandwidth to handle our tech debt

Reply:
Thanks for flagging this @djbarnwal !!! -- since you know this is unsafe for sure, I'm going to remove this cast before merging